### PR TITLE
29-Add-images-and-alt-texts

### LINKED ISSUE #29: Add seAp images
## Description
Each story has 4 accompanying illustrations, which can be found here:
https://drive.google.com/drive/folders/1GlVSZgxl6drg9_o8tXP4g41TeLd1pgT4?usp=sharing

## Plan
- [x] Get sign off from seAp on the final illustrations
- [x] Resize `convert [filename] -resize 2500 story[n]-image[n].jpg`
- [x] Compress with https://tinyjpg.com/
- [x] Add illustrations to the correct stories
- [x] Crop to square and add thumbnails for each story
- [x] Add alt text

## Checklist for each story
- [ ] Story 1
- [ ] Story 2
- [ ] Story 3
- [ ] Story 4
- [ ] Thumbnails for 1,2,3 and 4

## Acceptance criteria
- [ ] Stories have a thumbnail and 4 images
- [ ] The images have alt text that matches the copy doc https://docs.google.com/spreadsheets/d/1aFDhogqKhC2AogDuwdOVTlOytb3NxbsYjOaka43ssbw/edit?disco=AAAAEBebCik&usp=gmail_thread&ts=5df9f686
- [ ] There is a homepage banner image
- [ ] The homepage banner image is correctly pathed in the meta info

### PR DESCRIPTION
**Pull request by AlinChindea**
Fixes #29

## Description

- Add processed images to story cards
- Add provisional alt text for the images

@neontribe/ca-seap

### CODE REVIEW

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -515,7 +515,7 @@ brandCopy key =
                 me which key things I wanted to remember to tell them."""
 
         StoryOne3ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of a woman holding various stationery items"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Would "Illustration of a happy woman holding a box of tissues and letter writing materials" work?

File: src/Copy/BrandCopy.elm [hunk 2]
```diff
@@ -564,7 +564,7 @@ brandCopy key =
                 turned out I had appendicitis."""
 
         StoryTwo2ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of a nurse speaking to a woman vices"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Maybe "Illustration of a nurse speaking to a woman about sex and drinking" or "...speaking to a woman about vices"

File: src/Copy/BrandCopy.elm [hunk 3]
```diff
@@ -628,7 +628,7 @@ brandCopy key =
                 person to help me."""
 
         StoryThree2ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of 4 persons sitting next to each other, hands folded, with their back to a woman with a phone next to her"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Consider something like "Illustration of 4 persons sitting next to each other, arms folded, with their backs to a woman sitting with a phone and her hands in the air like she's given up"

File: src/Copy/BrandCopy.elm [hunk 4]
```diff
@@ -642,7 +642,7 @@ brandCopy key =
                 it was being logged."""
 
         StoryThree3ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of a woman talking to a man taking notes and listening to her with two boats in the background, one titled apology and the other complaint"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Maybe "Illustration of a shrugging woman, talking to a man who is taking notes and listening to her with two boats in the background, one titled apology and the other complaint"

File: src/Copy/BrandCopy.elm [hunk 5]
```diff
@@ -655,7 +655,7 @@ brandCopy key =
                 It meant I could move on."""
 
         StoryThree4ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of a woman on a couch holding a pillow with sorry written on it"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: hugging rather than holding?

File: src/Copy/BrandCopy.elm [hunk 6]
```diff
@@ -704,7 +704,7 @@ brandCopy key =
                 I was waiting."""
 
         StoryFour3ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of a pencil, paper, a booklet and a beverage on a desk"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: leaflet ratehr than booklet? hot beverage?

**Review by katjam — COMMENTED** [review]
I'd like @roseb04 to take a second look at the copy but looks good to me.

File: src/Copy/BrandCopy.elm (hunk 2 above)
Comment (AlinChindea) [reply src/Copy/BrandCopy.elm]: "Illustration of a nurse speaking to a woman about contraception and drinking" - fits better with the image and the quote, I believe.

**Comment by AlinChindea** [comment]
> I'd like @roseb04 to take a second look at the copy but looks good to me.

@roseb04 if you can check out this branch and have a look at:

1. choice of images for thumbnails
2. alt text for both thumbnails and in-story images

that would be wonderful of you! :sun_with_face: :sunflower:

**Comment by rosejbon** [comment]
Looks perfect, thank you @AlinChindea I super appreciate you writing those alt texts! I am happy with those, good to go.

Punch it :D

**Comment by AlinChindea** [comment]
> Looks perfect, thank you @AlinChindea I super appreciate you writing those alt texts! I am happy with those, good to go.
> 
> Punch it :D

@roseb04 am happy to hear but I need @katjam to approve this PR first :dancing_men: :dancing_women:

**Review by katjam — APPROVED** [review]
Also looks good to me. I wasn't blocking. @roseb04 's word good enough. :1st_place_medal:

**Comment by katjam** [comment]
Though - to be fair, @roseb04 hadn;t officially approved either. Green tick.

**Review by rosejbon — APPROVED** [review]
:shipit:

**Pull request merged**